src/documentation/pages/components-wip/footer/examples/_internal-centurylink.vue

```diff
@@ -0,0 +1,85 @@
+<template lang="pug">
+  <ComponentExample title="Internal" id="internal-centurylink" :tabs="exampleTabs">
```

Comment:
It does not look like the same footer.

![image](https://user-images.githubusercontent.com/86958774/176477935-db577a8c-c119-460a-be09-bf60d41305f0.png)

Reply:
@josecarlosgonzalezv I just reviewed with the current prod version. It looks the same. Century link has the lighter background where as Lumen has dark background for footers. Can you double check?